lib/phraseapp_updater/phraseapp_api.rb

```diff
@@ -99,35 +100,56 @@ def download_files(locales, skip_unverified:)
     # Empirically, PhraseApp fails to parse the uploaded files when uploaded in
     # parallel. Give it a better chance by uploading them one at a time.
     def upload_files(locale_files, default_locale:)
-      is_default = ->(l) { l.locale_name == default_locale }
+      locale_files = locale_files.sort_by(&:locale_name)
+      default_locale_file = locale_files.detect { |l| l.locale_name == default_locale }
+      locale_files.delete(default_locale_file) if default_locale_file
 
-      # Ensure the locales all exist
-      STDERR.puts('Creating locales')
       known_locales = fetch_locales.index_by(&:name)
+
+      if default_locale_file
+        unless known_locales.has_key?(default_locale_file.locale_name)
+          STDERR.puts("Creating default locale (#{default_locale_file})")
+          create_locale(default_locale_file.locale_name, default: true)
+        end
+
+        STDERR.puts("Uploading default locale (#{default_locale_file})")
+        upload_id = upload_file(default_locale_file)
+
+        successful_default_upload = verify_uploads({ upload_id => default_locale_file })
+      else
+        STDERR.puts("No upload for default locale (#{default_locale})")
+      end
```

Comment:
I would document this block about the motivation for separating the default locale from the other locale files.

```
# The default locale file is being handled separately and first. Phraseapp appears to have 
# behaviors for resolving key conflicts (pluralized versus non pluralized) by using precedence.
```